templates/accounts/registration.html

```diff
@@ -3,26 +3,26 @@
+    <p>Members are the backbone of our community.</p>
+    <p>As a member, you can provide the extra insights that bring our resources alive, and help our knowledge base grow by uploading your own resources about innovation and best practice in campaigning and organising.</p>
+    <p>Anyone can become a member of The Campaigns Library. You just have to register, by filling out the form above. Once your registration is received, the Librarian will have a look and make sure your account is set up correctly. Then you’ll be up and running!</p>
+    <p>Once you are a member, you’ll be able to log in and:</p>
     <ul>
       <li>Let the community know which resources have worked best for you;</li>
       <li>Leave feedback and tips for others, and join in discussions via comments;</li>
-      <li>Connect with other members;&nbsp;</li>
+      <li>Connect with other members;</li>
       <li>Receive updates, news and tips from the Librarian; and</li>
       <li>Upload your own resources, helping our Workspace grow.</li>
     </ul>
-    <p>When you upload a resource to the&nbsp;Workspace, the Librarian may reach out and ask if the Commons Library can include your resource in its collection, available to the public at&nbsp;<a href="https://commonslibrary.org/">https://commonslibrary.org/</a></p>
-    <p>The Commons Library Workspace website is also a resource in and of itself - our own platform and information architecture system is fully open source. You can create your own group on the platform with its own Workspace as your own resource management tool. Note that there are <a href="/terms/">Terms and Conditions of Use</a> applicable.</p>
-    <p>If you already know of any existing group (for example, one which has already been created for your work colleagues) that you wish to join, please indicate this when you register, and the Librarian will confirm your access to the relevant Workspaces. If you would like to start your own group and associated Workspace, or to upload resources to the public Workspace site&nbsp;on behalf of a group, get in touch with the Librarian at <a href="mailto:librarian@commonslibrary.org">librarian@commonslibrary.org</a></p>
+    <p>The Campaigns Library is also a resource in and of itself - our own platform and information architecture system is fully open source. <strong>Your organisation can use the Library as its own resource management tool, and maintain either a public or private collection.</strong> Note that there are <a href="/terms/">Terms and Conditions of Use</a> applicable.</p>
+    <p>If you already belong to an Organisation with a collection please indicate this when you register, and the Librarian will confirm your access to the relevant Organisations.</p>
+    <p>If your organisation would like its own private collection, or to upload resources to the public Library, get in touch with the Librarian at <a href="mailto:librarian@the-open.org">librarian@the-open.org</a></p>
   </div>
 {% endblock main_content %}
```

Comment:
would prefer to have long paragraphs broken up, but not a biggy !

Reply:
yeah i'll fix that i've been inconsistent with this.